docs/DEVELOPING.md

```diff
@@ -82,11 +82,13 @@ The connector uses the Elastic ingest attachment processor plugin for extracting
 
 ### E2E Tests
 
-The framework allows users to test the connector end to end. To perform e2e test for Network Drive connector, run the following make command:
+The end to end test is usually performed by developers after the functional and system testing is completed. The framework allows users to test the connector end to end. To perform e2e test for Network Drive connector, run the following make command:
```

Comment:
```suggestion
The framework provides a way to test ingestion through a connector against a real data source. This is called a functional test. To execute a functional test for the Network Drive connector, run the following command:
```

I would suggest we don't use several different terms and stick to one. In this case I would suggest to stick to "functional test". cc @tarekziade WDYT?